frontend/src/App.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { MantineProvider, Button } from '@mantine/core';
+
+export default function App() {
+    return (
+        <MantineProvider theme={{ fontFamily: "Greycliff CF, sans-serif" }}>
+            <Button variant="gradient">Does something</Button>
```

Comment:
For now Instead of Button, place here just a text that says something like "Welcome to Weather AI Website"